packages/fxa-settings/src/components/TermsPrivacyAgreement/index.test.tsx

```diff
@@ -51,17 +70,12 @@ it('renders component as expected for Monitor clients', () => {
   renderWithLocalizationProvider(<TermsPrivacyAgreement isMonitorClient />);
   // testAllL10n(screen, bundle);
 
-  const linkElements: HTMLElement[] = screen.getAllByRole('link');
+  testSubscriptionTos();
+});
 
-  expect(linkElements).toHaveLength(4);
-  expect(linkElements[0]).toHaveAttribute(
-    'href',
-    'https://www.mozilla.org/about/legal/terms/subscription-services/'
-  );
-  expect(linkElements[1]).toHaveAttribute(
-    'href',
-    'https://www.mozilla.org/privacy/subscription-services/'
-  );
-  expect(linkElements[2]).toHaveAttribute('href', '/legal/terms');
-  expect(linkElements[3]).toHaveAttribute('href', '/legal/privacy');
+it('renders component as expected when service=relay', () => {
+  renderWithLocalizationProvider(<TermsPrivacyAgreement isDesktopRelay />);
+  // testAllL10n(screen, bundle);
```

Comment:
nit: we could probably remove this commented out code, it's unclear when we'll get to setting up these tests

Reply:
True, but it's scattered throughout this test and other files so I think I will leave this for now to avoid another CI run 😅